worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -372,6 +408,49 @@ public Void call() throws Exception {
         }
     }
 
+    private static class SchematicShareTask implements Callable<URL> {
+        private final Actor actor;
+        private final ClipboardFormat format;
+        private final ClipboardHolder holder;
+
+        SchematicShareTask(Actor actor, ClipboardFormat format, ClipboardHolder holder) {
+            this.actor = actor;
+            this.format = format;
+            this.holder = holder;
+        }
+
+        @Override
+        public URL call() throws Exception {
+            Clipboard clipboard = holder.getClipboard();
+            Transform transform = holder.getTransform();
+            Clipboard target;
+
+            // If we have a transform, bake it into the copy
+            if (transform.isIdentity()) {
+                target = clipboard;
+            } else {
+                FlattenedClipboardTransform result = FlattenedClipboardTransform.transform(clipboard, transform);
+                target = new BlockArrayClipboard(result.getTransformedRegion());
+                target.setOrigin(clipboard.getOrigin());
+                Operations.completeLegacy(result.copyTo(target));
+            }
+
+            ByteArrayOutputStream baos;
+
+            try (Closer closer = Closer.create()) {
+                baos = closer.register(new ByteArrayOutputStream());
+                BufferedOutputStream bos = closer.register(new BufferedOutputStream(baos));
+                ClipboardWriter writer = closer.register(format.getWriter(bos));
+                writer.write(target);
+            } catch (IOException e) {
+                throw new CommandException(TextComponent.of(e.getMessage()), e, ImmutableList.of());
+            }
+
+            EngineHubPaste pasteService = new EngineHubPaste();
+            return pasteService.paste(new String(Base64.getEncoder().encode(baos.toByteArray()), Charsets.UTF_8)).call();
```

Comment:
```suggestion
            return pasteService.paste(new String(Base64.getEncoder().encode(baos.toByteArray()), StandardCharsets.UTF_8)).call();
```
No need to use Guava, the JDK has this.